docs/api-forms.md

```diff
@@ -74,25 +74,25 @@ however, there will be validation checks.
 import { ValidatedForm } from 'uniforms'; // Or from the theme package.
 
 <ValidatedForm
-  onValidate={(model, error, callback) => {
+  onValidate={async (model, error) => {
     // You can either ignore validation error...
     if (omitValidation(model)) {
-      return callback(null);
+      return null;
     }
 
     // ...or any additional validation if an error is already there...
     if (error) {
-      return callback();
+      return;
```

Comment:
Frankly, I don't understand the comment above and what this example is about. Can you explain it, so maybe I can  rephrase the comment somehow?

Reply:
The point here is that sometimes you'd like to transform the error. For example, you may want to hide an error _X is required_ if the user is offline.